phpstan.neon.template

```diff
@@ -6,9 +6,10 @@ includes:
 
 parameters:
 	scanDirectories:
+		- {VENDOR_DIR}/civicrm/civicrm-core/api/
 		- {VENDOR_DIR}/civicrm/civicrm-core/Civi/
 		- {VENDOR_DIR}/civicrm/civicrm-core/CRM/
-		- {VENDOR_DIR}/civicrm/civicrm-core/api/
+		- {VENDOR_DIR}/civicrm/civicrm-core/ext/civi_contribute/
```

Comment:
This requires at least CiviCRM 5.63: https://github.com/civicrm/civicrm-core/commit/0e332d88b62ef2694c96f9b1bf3c34caceb01096